### PR TITLE
Implement `snapcraft define` for parts

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

File: snapcraft/internal/parts.py [hunk 1]
```diff
@@ -126,5 +128,20 @@ def update():
     _Update().execute()
 
 
+def define(part_name):
+    try:
+        remote_part = _RemoteParts().get_part(part_name, full=True)
+    except KeyError as e:
+        raise RuntimeError(
+            'Cannot find the part name {!r} in the cache. Please '
+            'consider going to https://wiki.ubunut.com/snapcraft/parts '
```
Comment (elopio) [inline snapcraft/internal/parts.py]: typo: ubunut.

File: snapcraft/internal/parts.py [hunk 2]
```diff
@@ -126,5 +128,20 @@ def update():
...
+            'to add it.') from e
+    print('Maintainer: {!r}'.format(remote_part.pop('maintainer')))
+    print('Description: {!r}'.format(remote_part.pop('description')))
+    print('')
```
Comment (elopio) [inline snapcraft/internal/parts.py]: log instead of print?

**Comment by elopio** [comment]
lgtm. Just a typo and a question.

File: snapcraft/internal/parts.py (hunk 2 above)
Comment (sergiusens) [inline snapcraft/internal/parts.py]: log seems weird here though

**Pull request merged**